student.rb

```diff
@@ -1,10 +1,12 @@
 # create a student class 
+require_relative 'Person'
```

Comment:
- [ ] For some reason, your console app was not working when I start it. On closer inspection, I saw that the issue is how you are requiring them. Below I have included for you the correct one.

```
require_relative './person'
```